docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-installing-google-tag-manager.md

```diff
@@ -2,70 +2,49 @@
 title: "Installing Google Tag Manager"
 slug: "vtex-io-documentation-installing-google-tag-manager"
 hidden: false
-createdAt: "2022-02-01T13:34:01.638Z"
-updatedAt: "2023-05-03T11:17:44.697Z"
-excerpt: "Step-by-step on how to install the Google Tag Manager app in your VTEX Admin."
+createdAt: "2022-02-01t13:34:01.638z"
+updatedAt: "2022-12-13T20:17:44.697Z"
+excerpt: "Steps to install the Google Tag Manager app in your VTEX Admin."
 category: "Storefront Development"
+seeAlso:
+  - "/docs/guides/vtex-io-documentation-setting-up-google-tag-manager"
 ---
 
```

Comment:
```suggestion

This guide provides a step-by-step guide on installing and configuring Google Tag Manager (GTM) on Store Framework stores. The guide also highlights some restrictions to avoid performance issues and unpredictable behavior, including blocklists for custom HTML tags and variables.
```